pkg/pillar/hypervisor/kvm.go

```diff
@@ -583,6 +593,12 @@ func (ctx kvmContext) PCIRelease(long string) error {
 	overrideFile := sysfsPciDevices + long + "/driver_override"
 	unbindFile := sysfsPciDevices + long + "/driver/unbind"
 
+	//Write Empty string, to clear driver_override for the device
+	if err := ioutil.WriteFile(overrideFile, []byte(""), 0644); err != nil {
```

Comment:
Extreme nit but you should be able to pass an empty byte slice without converting from a zero length string.
[]byte{} or something like that.